### DISCUSSION TITLE
Open5gs 5G network deployment over istio

### DISCUSSION THREAD

**Discussion by jmbenol** [istio/istio#50675] [Q&A]
Hello,

We have deployed 5G openverso/open5gs network on kubernetes cluster.

In 5G network, UPF component provides end user UE navigation. It is responsible for decapsulating GTP traffic received from UE and routing traffic to destination.

When istio is enabled, seems like Istio envoy in UPF doesn't route UE traffic outside the pod. This is a network trace taken in UPF:
![image](https://github.com/istio/istio/assets/62904263/8ac10ed7-3f08-4fad-b0db-b8b6bf925782)

When excluding ports 80/443 in Istio, everything works fine meaning UE can browse Internet. But we would like to avoid it in order to gain Istio observability on those ports

I know this is very specific to 5G technology and maybe GTP protocol. In any case, I would appreciate if somebody could give a hint.

Thanks in advance,

Best Regards,
José Miguel

**Comment by LSKhappychild** [comment]
Having similar issue for integrating istio with OAI 5G CN. Is there any progress so far?
